lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -2531,6 +2531,23 @@ LogicalResult RefDefineOp::verify() {
   return success();
 }
 
+LogicalResult PropAssignOp::verify() {
```

Comment:
Might be good to share code w/RefDefineOp's verifier (modulo blacklisted destination ops, not sure what story is for properties--just remember only-has-one-driver gets a little less trivial to check once aggregates enter the scene, particularly if you're allowed to have "aliasing" sub-accesses to same element (refs don't have this problem, for now anyway :smile:).